src/main/scala/com/evolutiongaming/cluster/ActorRefExtension.scala

```diff
@@ -0,0 +1,5 @@
+package com.evolutiongaming.cluster
+
+import akka.actor.{ActorRef, Extension}
+
+class ActorRefExtension(val ref: ActorRef) extends Extension
```

Comment:
Oh I see you've implemented the extension but the name of `MappedAllocationStrategyDDProxy` did not match to `ActorRefExtension `, thus I missed it and was confused :)